src/client.ts

```
@@ -657,6 +678,35 @@ export class BikeTagClient {
+    payload?: RequireAtLeastOne<getGamePayload> | string,
+    opts?: RequireAtLeastOne<Credentials>
+  ): Promise<BikeTagApiResponse<Game[]>> {
+    const { client, options, api, source } = this.getClientAdapter(
+      payload,
+      opts,
+      DataTypes.game,
+      'getGame'
+    )
+    const clientMethod = api.getGame
+    /// If we remove the game we are looking for, we get ALL games
+    delete options.game
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)